g/grpc-gateway/grpc-gateway_ubi_8.4.sh

```diff
@@ -0,0 +1,72 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : grpc-gateway 
+# Version       : v2.6.0
+# Source repo   : https://github.com/grpc-ecosystem/grpc-gateway  
+# Tested on     : UBI 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Nagesh Tarale <Nagesh.Tarale@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
```

Comment:
Can we set the latest tested version (2.6.0??) as the default

Reply:
Yes 2.6.0 is latest version which is pointing to Master. By default it will be master.

Reply:
Logs:
------
[root@406a14c090d5 /]# ./grpc-gateway_ubi_8.4.sh
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 0:12:56 ago on Tue Nov  2 07:40:53 2021.
Dependencies resolved.
Nothing to do.
Complete!
Updating Subscription Management repositories.
Unable to read consumer identity

This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.

Last metadata expiration check: 0:12:58 ago on Tue Nov  2 07:40:53 2021.
Package golang-1.15.14-2.module+el8.4.0+12542+e3fec473.ppc64le is already installed.
Package git-2.27.0-1.el8.ppc64le is already installed.
Package wget-1.19.5-10.el8.ppc64le is already installed.
Dependencies resolved.
Nothing to do.
Complete!
Cloning into 'grpc-gateway'...
remote: Enumerating objects: 13918, done.
remote: Counting objects: 100% (449/449), done.
remote: Compressing objects: 100% (321/321), done.
remote: Total 13918 (delta 291), reused 159 (delta 110), pack-reused 13469
Receiving objects: 100% (13918/13918), 5.76 MiB | 15.90 MiB/s, done.
Resolving deltas: 100% (9850/9850), done.
Already on 'master'
Your branch is up to date with 'origin/master'.
 found to checkout.........
Build command is successful
Build successful
TESTING IN PROGRESS.....
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/abe [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/echo        [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/generateunboundmethods      [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/responsebody        [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/clients/unannotatedecho     [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/cmd/example-gateway-server  [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/cmd/example-grpc-server     [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/gateway     [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/helloworld  [no test files]
=== RUN   TestEchoClient
--- PASS: TestEchoClient (0.00s)
=== RUN   TestEchoBodyClient
--- PASS: TestEchoBodyClient (0.00s)
=== RUN   TestAbitOfEverythingClient
--- PASS: TestAbitOfEverythingClient (0.00s)
=== RUN   TestUnannotatedEchoClient
--- PASS: TestUnannotatedEchoClient (0.00s)
=== RUN   TestUnannotatedEchoBodyClient
--- PASS: TestUnannotatedEchoBodyClient (0.00s)
=== RUN   TestEcho
=== RUN   TestEcho/v1
=== RUN   TestEcho/v2
--- PASS: TestEcho (0.01s)
    --- PASS: TestEcho/v1 (0.01s)
    --- PASS: TestEcho/v2 (0.01s)
=== RUN   TestEchoUnauthorized
--- PASS: TestEchoUnauthorized (0.00s)
=== RUN   TestEchoPatch
--- PASS: TestEchoPatch (0.00s)
=== RUN   TestForwardResponseOption
--- PASS: TestForwardResponseOption (0.01s)
=== RUN   TestABE
--- PASS: TestABE (0.04s)
=== RUN   TestABEPatch
--- PASS: TestABEPatch (0.01s)
=== RUN   TestABEPatchBody
=== RUN   TestABEPatchBody/with_fieldmask_provided
=== RUN   TestABEPatchBody/with_empty_fieldmask
=== RUN   TestABEPatchBody/with_nil_fieldmask
--- PASS: TestABEPatchBody (0.01s)
    --- PASS: TestABEPatchBody/with_fieldmask_provided (0.00s)
    --- PASS: TestABEPatchBody/with_empty_fieldmask (0.00s)
    --- PASS: TestABEPatchBody/with_nil_fieldmask (0.00s)
=== RUN   TestTimeout
--- PASS: TestTimeout (0.01s)
=== RUN   TestPostWithEmptyBody
--- PASS: TestPostWithEmptyBody (0.00s)
=== RUN   TestUnknownPath
--- PASS: TestUnknownPath (0.00s)
=== RUN   TestNotImplemented
--- PASS: TestNotImplemented (0.00s)
=== RUN   TestInvalidArgument
--- PASS: TestInvalidArgument (0.00s)
=== RUN   TestResponseBody
=== RUN   TestResponseBody/unary_case
=== RUN   TestResponseBody/Response_strings
=== RUN   TestResponseBody/Empty_response_strings
=== RUN   TestResponseBody/Response_bodies
--- PASS: TestResponseBody (0.02s)
    --- PASS: TestResponseBody/unary_case (0.00s)
    --- PASS: TestResponseBody/Response_strings (0.00s)
    --- PASS: TestResponseBody/Empty_response_strings (0.00s)
    --- PASS: TestResponseBody/Response_bodies (0.00s)
=== RUN   TestResponseBodyStream
=== RUN   TestResponseBodyStream/stream_case
--- PASS: TestResponseBodyStream (0.00s)
    --- PASS: TestResponseBodyStream/stream_case (0.00s)
=== RUN   TestRequestQueryParams
=== RUN   TestRequestQueryParams/get_url_query_values
=== RUN   TestRequestQueryParams/get_nested_enum_url_parameter
=== RUN   TestRequestQueryParams/post_url_query_values
=== RUN   TestRequestQueryParams/post_form_and_url_query_values
--- PASS: TestRequestQueryParams (0.00s)
    --- PASS: TestRequestQueryParams/get_url_query_values (0.00s)
    --- PASS: TestRequestQueryParams/get_nested_enum_url_parameter (0.00s)
    --- PASS: TestRequestQueryParams/post_url_query_values (0.00s)
    --- PASS: TestRequestQueryParams/post_form_and_url_query_values (0.00s)
=== RUN   TestRequestQueryParamsInProcessGateway
=== RUN   TestRequestQueryParamsInProcessGateway/get_url_query_values
=== RUN   TestRequestQueryParamsInProcessGateway/get_nested_enum_url_parameter
=== RUN   TestRequestQueryParamsInProcessGateway/post_url_query_values
=== RUN   TestRequestQueryParamsInProcessGateway/post_form_and_url_query_values
--- PASS: TestRequestQueryParamsInProcessGateway (0.00s)
    --- PASS: TestRequestQueryParamsInProcessGateway/get_url_query_values (0.00s)
    --- PASS: TestRequestQueryParamsInProcessGateway/get_nested_enum_url_parameter (0.00s)
    --- PASS: TestRequestQueryParamsInProcessGateway/post_url_query_values (0.00s)
    --- PASS: TestRequestQueryParamsInProcessGateway/post_form_and_url_query_values (0.00s)
=== RUN   TestNonStandardNames
=== RUN   TestNonStandardNames/Test_standard_update_method
    integration_test.go:2238: {"id":"foo", "Num":"1", "line_num":"42", "langIdent":"English", "STATUS":"good", "en_GB":"1", "no":"yes", "thing":{"subThing":{"sub_value":"hi"}}}
=== RUN   TestNonStandardNames/Test_update_method_using_json_names_in_message
    integration_test.go:2238: {"id":"foo", "Num":"1", "line_num":"42", "langIdent":"English", "STATUS":"good", "en_GB":"1", "no":"yes", "thing":{"subThing":{"sub_value":"hi"}}}
=== RUN   TestNonStandardNames/Test_standard_update_method_with_UseProtoNames:_false_marshaller_option
    integration_test.go:2238: {"id":"foo", "Num":"1", "lineNum":"42", "langIdent":"English", "STATUS":"good", "enGB":"1", "no":"yes", "thing":{"subThing":{"subValue":"hi"}}}
=== RUN   TestNonStandardNames/Test_update_method_using_json_names_in_message_with_UseProtoNames:_false_marshaller_option
    integration_test.go:2238: {"ID":"foo", "Num":"1", "LineNum":"42", "langIdent":"English", "status":"good", "En_GB":"1", "yes":"yes", "Thingy":{"SubThing":{"sub_Value":"hi"}}}
--- PASS: TestNonStandardNames (0.02s)
    --- PASS: TestNonStandardNames/Test_standard_update_method (0.00s)
    --- PASS: TestNonStandardNames/Test_update_method_using_json_names_in_message (0.00s)
    --- PASS: TestNonStandardNames/Test_standard_update_method_with_UseProtoNames:_false_marshaller_option (0.00s)
    --- PASS: TestNonStandardNames/Test_update_method_using_json_names_in_message_with_UseProtoNames:_false_marshaller_option (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/integration (cached)
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/proto/examplepb     [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/proto/pathenum      [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/proto/standalone    [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/proto/sub   [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/proto/sub2  [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/examples/internal/server      [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/internal/casing       [no test files]
=== RUN   TestParseRequest
=== RUN   TestParseRequest/Empty_input_should_produce_empty_output
=== RUN   TestParseRequest/Invalid_reader_should_produce_error
=== RUN   TestParseRequest/Invalid_proto_message_should_produce_error
--- PASS: TestParseRequest (0.00s)
    --- PASS: TestParseRequest/Empty_input_should_produce_empty_output (0.00s)
    --- PASS: TestParseRequest/Invalid_reader_should_produce_error (0.00s)
    --- PASS: TestParseRequest/Invalid_proto_message_should_produce_error (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/internal/codegenerator        (cached)
=== RUN   TestLoadGrpcAPIServiceFromYAMLInvalidType
--- PASS: TestLoadGrpcAPIServiceFromYAMLInvalidType (0.00s)
=== RUN   TestLoadGrpcAPIServiceFromYAMLSingleRule
--- PASS: TestLoadGrpcAPIServiceFromYAMLSingleRule (0.00s)
=== RUN   TestLoadGrpcAPIServiceFromYAMLRejectInvalidYAML
--- PASS: TestLoadGrpcAPIServiceFromYAMLRejectInvalidYAML (0.00s)
=== RUN   TestLoadGrpcAPIServiceFromYAMLMultipleWithAdditionalBindings
--- PASS: TestLoadGrpcAPIServiceFromYAMLMultipleWithAdditionalBindings (0.00s)
=== RUN   TestLoadOpenAPIConfigFromYAMLRejectInvalidYAML
--- PASS: TestLoadOpenAPIConfigFromYAMLRejectInvalidYAML (0.00s)
=== RUN   TestLoadOpenAPIConfigFromYAML
--- PASS: TestLoadOpenAPIConfigFromYAML (0.00s)
=== RUN   TestLoadFile
--- PASS: TestLoadFile (0.00s)
=== RUN   TestLoadFileNestedPackage
--- PASS: TestLoadFileNestedPackage (0.00s)
=== RUN   TestLoadFileWithDir
--- PASS: TestLoadFileWithDir (0.00s)
=== RUN   TestLoadFileWithoutPackage
--- PASS: TestLoadFileWithoutPackage (0.00s)
=== RUN   TestLoadFileWithMapping
--- PASS: TestLoadFileWithMapping (0.00s)
=== RUN   TestLoadFileWithPackageNameCollision
--- PASS: TestLoadFileWithPackageNameCollision (0.00s)
=== RUN   TestLoadFileWithIdenticalGoPkg
--- PASS: TestLoadFileWithIdenticalGoPkg (0.00s)
=== RUN   TestLookupMsgWithoutPackage
--- PASS: TestLookupMsgWithoutPackage (0.00s)
=== RUN   TestLookupMsgWithNestedPackage
--- PASS: TestLookupMsgWithNestedPackage (0.00s)
=== RUN   TestLoadWithInconsistentTargetPackage
--- PASS: TestLoadWithInconsistentTargetPackage (0.00s)
=== RUN   TestLoadOverriddenPackageName
--- PASS: TestLoadOverriddenPackageName (0.00s)
=== RUN   TestLoadWithStandalone
--- PASS: TestLoadWithStandalone (0.00s)
=== RUN   TestUnboundExternalHTTPRules
--- PASS: TestUnboundExternalHTTPRules (0.00s)
=== RUN   TestRegisterOpenAPIOptions
=== RUN   TestRegisterOpenAPIOptions/handle_nil_options
=== RUN   TestRegisterOpenAPIOptions/successfully_add_options_if_referenced_entity_exists
=== RUN   TestRegisterOpenAPIOptions/reject_fully_qualified_names_with_leading_"."
=== RUN   TestRegisterOpenAPIOptions/error_if_file_does_not_exist
=== RUN   TestRegisterOpenAPIOptions/error_if_method_does_not_exist
=== RUN   TestRegisterOpenAPIOptions/error_if_message_does_not_exist
=== RUN   TestRegisterOpenAPIOptions/error_if_service_does_not_exist
=== RUN   TestRegisterOpenAPIOptions/error_if_field_does_not_exist
--- PASS: TestRegisterOpenAPIOptions (0.00s)
    --- PASS: TestRegisterOpenAPIOptions/handle_nil_options (0.00s)
    --- PASS: TestRegisterOpenAPIOptions/successfully_add_options_if_referenced_entity_exists (0.00s)
    --- PASS: TestRegisterOpenAPIOptions/reject_fully_qualified_names_with_leading_"." (0.00s)
    --- PASS: TestRegisterOpenAPIOptions/error_if_file_does_not_exist (0.00s)
    --- PASS: TestRegisterOpenAPIOptions/error_if_method_does_not_exist (0.00s)
    --- PASS: TestRegisterOpenAPIOptions/error_if_message_does_not_exist (0.00s)
    --- PASS: TestRegisterOpenAPIOptions/error_if_service_does_not_exist (0.00s)
    --- PASS: TestRegisterOpenAPIOptions/error_if_field_does_not_exist (0.00s)
=== RUN   TestExtractServicesSimple
--- PASS: TestExtractServicesSimple (0.00s)
=== RUN   TestExtractServicesWithoutAnnotation
--- PASS: TestExtractServicesWithoutAnnotation (0.00s)
=== RUN   TestExtractServicesGenerateUnboundMethods
--- PASS: TestExtractServicesGenerateUnboundMethods (0.00s)
=== RUN   TestExtractServicesCrossPackage
--- PASS: TestExtractServicesCrossPackage (0.00s)
=== RUN   TestExtractServicesWithBodyPath
--- PASS: TestExtractServicesWithBodyPath (0.00s)
=== RUN   TestExtractServicesWithPathParam
--- PASS: TestExtractServicesWithPathParam (0.00s)
=== RUN   TestExtractServicesWithAdditionalBinding
--- PASS: TestExtractServicesWithAdditionalBinding (0.00s)
=== RUN   TestExtractServicesWithError
    services_test.go:1059: no message found: StringMessage
    services_test.go:1059: no field "bool" found in StringMessage
    services_test.go:1059: no field "bool" found in StringMessage
    services_test.go:1059: not an aggregate type: mid in mid.bool
    services_test.go:1059: cannot use path parameter in client streaming
    services_test.go:1059: must not set request body when http method is GET: Echo
    services_test.go:1059: must not set request body when http method is DELETE except allow_delete_body option is true: RemoveResource
    services_test.go:1059: no message found: StringMessage
    services_test.go:1059: ExampleService.Echo: string is a protobuf message type. Protobuf message types cannot be used as path parameters, use a scalar value type (such as string) instead
--- PASS: TestExtractServicesWithError (0.00s)
=== RUN   TestResolveFieldPath
--- PASS: TestResolveFieldPath (0.00s)
=== RUN   TestExtractServicesWithDeleteBody
    services_test.go:1348: <nil>
    services_test.go:1348: must not set request body when http method is DELETE except allow_delete_body option is true: RemoveResource
--- PASS: TestExtractServicesWithDeleteBody (0.00s)
=== RUN   TestCauseErrorWithPathParam
--- PASS: TestCauseErrorWithPathParam (0.00s)
=== RUN   TestOptionalProto3URLPathMappingError
--- PASS: TestOptionalProto3URLPathMappingError (0.00s)
=== RUN   TestGoPackageStandard
--- PASS: TestGoPackageStandard (0.00s)
=== RUN   TestGoPackageString
--- PASS: TestGoPackageString (0.00s)
=== RUN   TestFieldPath
--- PASS: TestFieldPath (0.00s)
=== RUN   TestGoType
--- PASS: TestGoType (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/internal/descriptor   (cached)
?       github.com/grpc-ecosystem/grpc-gateway/v2/internal/descriptor/apiconfig [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/internal/descriptor/openapiconfig     [no test files]
?       github.com/grpc-ecosystem/grpc-gateway/v2/internal/generator    [no test files]
=== RUN   TestCompile
--- PASS: TestCompile (0.00s)
=== RUN   TestTokenize
--- PASS: TestTokenize (0.00s)
=== RUN   TestParseSegments
--- PASS: TestParseSegments (0.00s)
=== RUN   TestParseSegmentsWithErrors
--- PASS: TestParseSegmentsWithErrors (0.00s)
=== RUN   TestTemplateStringer
--- PASS: TestTemplateStringer (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/internal/httprule     (cached)
=== RUN   TestParseFlagsEmptyNoPanic
--- PASS: TestParseFlagsEmptyNoPanic (0.00s)
=== RUN   TestParseFlags
--- PASS: TestParseFlags (0.00s)
=== RUN   TestParseFlagsMultiple
--- PASS: TestParseFlagsMultiple (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway       (cached)
=== RUN   TestGenerator_Generate
--- PASS: TestGenerator_Generate (0.00s)
=== RUN   TestApplyTemplateHeader
--- PASS: TestApplyTemplateHeader (0.00s)
=== RUN   TestApplyTemplateRequestWithoutClientStreaming
--- PASS: TestApplyTemplateRequestWithoutClientStreaming (0.00s)
=== RUN   TestApplyTemplateRequestWithClientStreaming
--- PASS: TestApplyTemplateRequestWithClientStreaming (0.00s)
=== RUN   TestApplyTemplateInProcess
--- PASS: TestApplyTemplateInProcess (0.00s)
=== RUN   TestAllowPatchFeature
--- PASS: TestAllowPatchFeature (0.00s)
=== RUN   TestIdentifierCapitalization
--- PASS: TestIdentifierCapitalization (0.00s)
=== RUN   TestDuplicatePaths
--- PASS: TestDuplicatePaths (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway/internal/gengateway   (cached)
=== RUN   TestParseReqParam
=== RUN   TestParseReqParam/Test_0
=== RUN   TestParseReqParam/Test_1
=== RUN   TestParseReqParam/Test_2
=== RUN   TestParseReqParam/Test_3
=== RUN   TestParseReqParam/Test_4
=== RUN   TestParseReqParam/Test_5
=== RUN   TestParseReqParam/Test_6
=== RUN   TestParseReqParam/Test_7
=== RUN   TestParseReqParam/Test_8
=== RUN   TestParseReqParam/Test_9
=== RUN   TestParseReqParam/Test_10
=== RUN   TestParseReqParam/Test_11
=== RUN   TestParseReqParam/Test_12
--- PASS: TestParseReqParam (0.00s)
    --- PASS: TestParseReqParam/Test_0 (0.00s)
    --- PASS: TestParseReqParam/Test_1 (0.00s)
    --- PASS: TestParseReqParam/Test_2 (0.00s)
    --- PASS: TestParseReqParam/Test_3 (0.00s)
    --- PASS: TestParseReqParam/Test_4 (0.00s)
    --- PASS: TestParseReqParam/Test_5 (0.00s)
    --- PASS: TestParseReqParam/Test_6 (0.00s)
    --- PASS: TestParseReqParam/Test_7 (0.00s)
    --- PASS: TestParseReqParam/Test_8 (0.00s)
    --- PASS: TestParseReqParam/Test_9 (0.00s)
    --- PASS: TestParseReqParam/Test_10 (0.00s)
    --- PASS: TestParseReqParam/Test_11 (0.00s)
    --- PASS: TestParseReqParam/Test_12 (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2  (cached)
=== RUN   TestCycle
--- PASS: TestCycle (0.00s)
=== RUN   TestNaming
=== RUN   TestNaming/fqn
=== RUN   TestNaming/legacy
=== RUN   TestNaming/simple
--- PASS: TestNaming (0.00s)
    --- PASS: TestNaming/fqn (0.00s)
    --- PASS: TestNaming/legacy (0.00s)
    --- PASS: TestNaming/simple (0.00s)
=== RUN   TestMessageToQueryParametersWithEnumAsInt
--- PASS: TestMessageToQueryParametersWithEnumAsInt (0.00s)
=== RUN   TestMessageToQueryParameters
--- PASS: TestMessageToQueryParameters (0.00s)
=== RUN   TestMessageToQueryParametersNoRecursive
--- PASS: TestMessageToQueryParametersNoRecursive (0.00s)
=== RUN   TestMessageToQueryParametersRecursive
--- PASS: TestMessageToQueryParametersRecursive (0.03s)
=== RUN   TestMessageToQueryParametersWithJsonName
--- PASS: TestMessageToQueryParametersWithJsonName (0.00s)
=== RUN   TestMessageToQueryParametersWellKnownTypes
--- PASS: TestMessageToQueryParametersWellKnownTypes (0.00s)
=== RUN   TestApplyTemplateSimple
--- PASS: TestApplyTemplateSimple (0.00s)
=== RUN   TestApplyTemplateMultiService
--- PASS: TestApplyTemplateMultiService (0.00s)
=== RUN   TestApplyTemplateOverrideOperationID
=== RUN   TestApplyTemplateOverrideOperationID/verify_override_via_method_option
=== RUN   TestApplyTemplateOverrideOperationID/verify_override_options_annotations
--- PASS: TestApplyTemplateOverrideOperationID (0.00s)
    --- PASS: TestApplyTemplateOverrideOperationID/verify_override_via_method_option (0.00s)
    --- PASS: TestApplyTemplateOverrideOperationID/verify_override_options_annotations (0.00s)
=== RUN   TestApplyTemplateExtensions
=== RUN   TestApplyTemplateExtensions/verify_template_options_set_via_proto_options
=== RUN   TestApplyTemplateExtensions/verify_template_options_set_via_annotations
--- PASS: TestApplyTemplateExtensions (0.00s)
    --- PASS: TestApplyTemplateExtensions/verify_template_options_set_via_proto_options (0.00s)
    --- PASS: TestApplyTemplateExtensions/verify_template_options_set_via_annotations (0.00s)
=== RUN   TestApplyTemplateHeaders
=== RUN   TestApplyTemplateHeaders/verify_template_options_set_via_proto_options
--- PASS: TestApplyTemplateHeaders (0.00s)
    --- PASS: TestApplyTemplateHeaders/verify_template_options_set_via_proto_options (0.00s)
=== RUN   TestValidateHeaderType
--- PASS: TestValidateHeaderType (0.00s)
=== RUN   TestValidateDefaultValueType
--- PASS: TestValidateDefaultValueType (0.00s)
=== RUN   TestApplyTemplateRequestWithoutClientStreaming
--- PASS: TestApplyTemplateRequestWithoutClientStreaming (0.00s)
=== RUN   TestApplyTemplateRequestWithClientStreaming
--- PASS: TestApplyTemplateRequestWithClientStreaming (0.00s)
=== RUN   TestApplyTemplateRequestWithUnusedReferences
--- PASS: TestApplyTemplateRequestWithUnusedReferences (0.00s)
=== RUN   TestApplyTemplateRequestWithBodyQueryParameters
=== RUN   TestApplyTemplateRequestWithBodyQueryParameters/book_in_body
=== RUN   TestApplyTemplateRequestWithBodyQueryParameters/book_in_query
--- PASS: TestApplyTemplateRequestWithBodyQueryParameters (0.00s)
    --- PASS: TestApplyTemplateRequestWithBodyQueryParameters/book_in_body (0.00s)
    --- PASS: TestApplyTemplateRequestWithBodyQueryParameters/book_in_query (0.00s)
=== RUN   TestApplyTemplateProtobufAny
=== RUN   TestApplyTemplateProtobufAny/default_protobufAny_from_rpcStatus
=== RUN   TestApplyTemplateProtobufAny/protobufAny_referenced_in_message
=== RUN   TestApplyTemplateProtobufAny/protobufAny_referenced_in_message_with_default_errors_disabled
--- PASS: TestApplyTemplateProtobufAny (0.00s)
    --- PASS: TestApplyTemplateProtobufAny/default_protobufAny_from_rpcStatus (0.00s)
    --- PASS: TestApplyTemplateProtobufAny/protobufAny_referenced_in_message (0.00s)
    --- PASS: TestApplyTemplateProtobufAny/protobufAny_referenced_in_message_with_default_errors_disabled (0.00s)
=== RUN   TestTemplateWithJsonCamelCase
--- PASS: TestTemplateWithJsonCamelCase (0.00s)
=== RUN   TestTemplateWithoutJsonCamelCase
--- PASS: TestTemplateWithoutJsonCamelCase (0.00s)
=== RUN   TestTemplateToOpenAPIPath
--- PASS: TestTemplateToOpenAPIPath (0.00s)
=== RUN   TestResolveFullyQualifiedNameToOpenAPIName
--- PASS: TestResolveFullyQualifiedNameToOpenAPIName (0.00s)
=== RUN   TestFQMNtoOpenAPIName
--- PASS: TestFQMNtoOpenAPIName (0.00s)
=== RUN   TestSchemaOfField
--- PASS: TestSchemaOfField (0.01s)
=== RUN   TestRenderMessagesAsDefinition
=== RUN   TestRenderMessagesAsDefinition/no_OpenAPI_options
=== RUN   TestRenderMessagesAsDefinition/example_option
=== RUN   TestRenderMessagesAsDefinition/example_option_with_something_non-json
=== RUN   TestRenderMessagesAsDefinition/external_docs_option
=== RUN   TestRenderMessagesAsDefinition/JSONSchema_options
=== RUN   TestRenderMessagesAsDefinition/JSONSchema_options_from_registry
=== RUN   TestRenderMessagesAsDefinition/JSONSchema_with_required_properties
=== RUN   TestRenderMessagesAsDefinition/JSONSchema_with_excluded_fields
=== RUN   TestRenderMessagesAsDefinition/JSONSchema_with_required_properties_via_field_behavior
--- PASS: TestRenderMessagesAsDefinition (0.00s)
    --- PASS: TestRenderMessagesAsDefinition/no_OpenAPI_options (0.00s)
    --- PASS: TestRenderMessagesAsDefinition/example_option (0.00s)
    --- PASS: TestRenderMessagesAsDefinition/example_option_with_something_non-json (0.00s)
    --- PASS: TestRenderMessagesAsDefinition/external_docs_option (0.00s)
    --- PASS: TestRenderMessagesAsDefinition/JSONSchema_options (0.00s)
    --- PASS: TestRenderMessagesAsDefinition/JSONSchema_options_from_registry (0.00s)
    --- PASS: TestRenderMessagesAsDefinition/JSONSchema_with_required_properties (0.00s)
    --- PASS: TestRenderMessagesAsDefinition/JSONSchema_with_excluded_fields (0.00s)
    --- PASS: TestRenderMessagesAsDefinition/JSONSchema_with_required_properties_via_field_behavior (0.00s)
=== RUN   TestUpdateOpenAPIDataFromComments
=== RUN   TestUpdateOpenAPIDataFromComments/empty_comments
=== RUN   TestUpdateOpenAPIDataFromComments/set_field_to_read_only
=== RUN   TestUpdateOpenAPIDataFromComments/set_title
=== RUN   TestUpdateOpenAPIDataFromComments/set_description
=== RUN   TestUpdateOpenAPIDataFromComments/use_info_object
=== RUN   TestUpdateOpenAPIDataFromComments/multi_line_comment_with_title
=== RUN   TestUpdateOpenAPIDataFromComments/multi_line_comment_no_title
=== RUN   TestUpdateOpenAPIDataFromComments/multi_line_comment_with_summary_with_dot
=== RUN   TestUpdateOpenAPIDataFromComments/multi_line_comment_with_summary_no_dot
=== RUN   TestUpdateOpenAPIDataFromComments/multi_line_comment_with_summary_no_dot#01
=== RUN   TestUpdateOpenAPIDataFromComments/without_use_go_template
=== RUN   TestUpdateOpenAPIDataFromComments/error_with_use_go_template
=== RUN   TestUpdateOpenAPIDataFromComments/template_with_use_go_template
--- PASS: TestUpdateOpenAPIDataFromComments (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/empty_comments (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/set_field_to_read_only (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/set_title (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/set_description (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/use_info_object (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/multi_line_comment_with_title (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/multi_line_comment_no_title (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/multi_line_comment_with_summary_with_dot (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/multi_line_comment_with_summary_no_dot (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/multi_line_comment_with_summary_no_dot#01 (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/without_use_go_template (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/error_with_use_go_template (0.00s)
    --- PASS: TestUpdateOpenAPIDataFromComments/template_with_use_go_template (0.00s)
=== RUN   TestMessageOptionsWithGoTemplate
=== RUN   TestMessageOptionsWithGoTemplate/external_docs_option
=== RUN   TestMessageOptionsWithGoTemplate/external_docs_option#01
=== RUN   TestMessageOptionsWithGoTemplate/registered_OpenAPIOption
--- PASS: TestMessageOptionsWithGoTemplate (0.00s)
    --- PASS: TestMessageOptionsWithGoTemplate/external_docs_option (0.00s)
    --- PASS: TestMessageOptionsWithGoTemplate/external_docs_option#01 (0.00s)
    --- PASS: TestMessageOptionsWithGoTemplate/registered_OpenAPIOption (0.00s)
=== RUN   TestTemplateWithoutErrorDefinition
E1102 07:49:05.382237    3601 template.go:1296] no message found: Status
--- PASS: TestTemplateWithoutErrorDefinition (0.00s)
=== RUN   Test_getReservedJsonName
=== RUN   Test_getReservedJsonName/test_case_1:_single_dot_use_case
=== RUN   Test_getReservedJsonName/test_case_2:_single_dot_use_case_with_no_existing_field
=== RUN   Test_getReservedJsonName/test_case_3:_double_dot_use_case
=== RUN   Test_getReservedJsonName/test_case_4:_double_dot_use_case_with_a_not_existed_field
--- PASS: Test_getReservedJsonName (0.00s)
    --- PASS: Test_getReservedJsonName/test_case_1:_single_dot_use_case (0.00s)
    --- PASS: Test_getReservedJsonName/test_case_2:_single_dot_use_case_with_no_existing_field (0.00s)
    --- PASS: Test_getReservedJsonName/test_case_3:_double_dot_use_case (0.00s)
    --- PASS: Test_getReservedJsonName/test_case_4:_double_dot_use_case_with_a_not_existed_field (0.00s)
=== RUN   TestParseIncompleteSecurityRequirement
E1102 07:49:05.383018    3601 template.go:1296] no message found: Status
--- PASS: TestParseIncompleteSecurityRequirement (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2/internal/genopenapi      (cached)
?       github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2/options  [no test files]
=== RUN   TestFieldMaskFromRequestBody
=== RUN   TestFieldMaskFromRequestBody/empty
=== RUN   TestFieldMaskFromRequestBody/simple
=== RUN   TestFieldMaskFromRequestBody/NonStandardMessage
=== RUN   TestFieldMaskFromRequestBody/NonStandardMessageWithJSONNames
=== RUN   TestFieldMaskFromRequestBody/nested
=== RUN   TestFieldMaskFromRequestBody/struct
=== RUN   TestFieldMaskFromRequestBody/value
=== RUN   TestFieldMaskFromRequestBody/map
=== RUN   TestFieldMaskFromRequestBody/deeply-nested
=== RUN   TestFieldMaskFromRequestBody/complex
=== RUN   TestFieldMaskFromRequestBody/protobuf-any
--- PASS: TestFieldMaskFromRequestBody (0.01s)
    --- PASS: TestFieldMaskFromRequestBody/empty (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/simple (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/NonStandardMessage (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/NonStandardMessageWithJSONNames (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/nested (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/struct (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/value (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/map (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/deeply-nested (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/complex (0.00s)
    --- PASS: TestFieldMaskFromRequestBody/protobuf-any (0.00s)
=== RUN   TestFieldMaskRepeatedFieldsLast
=== RUN   TestFieldMaskRepeatedFieldsLast/map
=== RUN   TestFieldMaskRepeatedFieldsLast/slice
--- PASS: TestFieldMaskRepeatedFieldsLast (0.00s)
    --- PASS: TestFieldMaskRepeatedFieldsLast/map (0.00s)
    --- PASS: TestFieldMaskRepeatedFieldsLast/slice (0.00s)
=== RUN   TestFieldMaskErrors
=== RUN   TestFieldMaskErrors/object_under_scalar
--- PASS: TestFieldMaskErrors (0.00s)
    --- PASS: TestFieldMaskErrors/object_under_scalar (0.00s)
=== RUN   TestNewPattern
--- PASS: TestNewPattern (0.00s)
=== RUN   TestNewPatternWithWrongOp
--- PASS: TestNewPatternWithWrongOp (0.00s)
=== RUN   TestNewPatternWithStackUnderflow
--- PASS: TestNewPatternWithStackUnderflow (0.00s)
=== RUN   TestMatch
--- PASS: TestMatch (0.00s)
=== RUN   TestMatchWithBinding
--- PASS: TestMatchWithBinding (0.00s)
=== RUN   TestPatternString
--- PASS: TestPatternString (0.00s)
=== RUN   TestAnnotateContext_WorksWithEmpty
--- PASS: TestAnnotateContext_WorksWithEmpty (0.00s)
=== RUN   TestAnnotateContext_ForwardsGrpcMetadata
--- PASS: TestAnnotateContext_ForwardsGrpcMetadata (0.00s)
=== RUN   TestAnnotateContext_ForwardGrpcBinaryMetadata
--- PASS: TestAnnotateContext_ForwardGrpcBinaryMetadata (0.00s)
=== RUN   TestAnnotateContext_XForwardedFor
--- PASS: TestAnnotateContext_XForwardedFor (0.00s)
=== RUN   TestAnnotateContext_SupportsTimeouts
--- PASS: TestAnnotateContext_SupportsTimeouts (0.00s)
=== RUN   TestAnnotateContext_SupportsCustomAnnotators
--- PASS: TestAnnotateContext_SupportsCustomAnnotators (0.00s)
=== RUN   TestAnnotateIncomingContext_WorksWithEmpty
--- PASS: TestAnnotateIncomingContext_WorksWithEmpty (0.00s)
=== RUN   TestAnnotateIncomingContext_ForwardsGrpcMetadata
--- PASS: TestAnnotateIncomingContext_ForwardsGrpcMetadata (0.00s)
=== RUN   TestAnnotateIncomingContext_ForwardGrpcBinaryMetadata
--- PASS: TestAnnotateIncomingContext_ForwardGrpcBinaryMetadata (0.00s)
=== RUN   TestAnnotateIncomingContext_XForwardedFor
--- PASS: TestAnnotateIncomingContext_XForwardedFor (0.00s)
=== RUN   TestAnnotateIncomingContext_SupportsTimeouts
--- PASS: TestAnnotateIncomingContext_SupportsTimeouts (0.00s)
=== RUN   TestAnnotateIncomingContext_SupportsCustomAnnotators
--- PASS: TestAnnotateIncomingContext_SupportsCustomAnnotators (0.00s)
=== RUN   TestConvertTimestamp
=== RUN   TestConvertTimestamp/a_valid_RFC3339_timestamp
=== RUN   TestConvertTimestamp/a_valid_RFC3339_timestamp_without_double_quotation
=== RUN   TestConvertTimestamp/invalid_timestamp
=== RUN   TestConvertTimestamp/JSON_number
=== RUN   TestConvertTimestamp/JSON_bool
--- PASS: TestConvertTimestamp (0.00s)
    --- PASS: TestConvertTimestamp/a_valid_RFC3339_timestamp (0.00s)
    --- PASS: TestConvertTimestamp/a_valid_RFC3339_timestamp_without_double_quotation (0.00s)
    --- PASS: TestConvertTimestamp/invalid_timestamp (0.00s)
    --- PASS: TestConvertTimestamp/JSON_number (0.00s)
    --- PASS: TestConvertTimestamp/JSON_bool (0.00s)
=== RUN   TestConvertDuration
=== RUN   TestConvertDuration/a_valid_duration
=== RUN   TestConvertDuration/a_valid_duration_without_double_quotation
=== RUN   TestConvertDuration/invalid_duration
=== RUN   TestConvertDuration/JSON_number
=== RUN   TestConvertDuration/JSON_bool
--- PASS: TestConvertDuration (0.00s)
    --- PASS: TestConvertDuration/a_valid_duration (0.00s)
    --- PASS: TestConvertDuration/a_valid_duration_without_double_quotation (0.00s)
    --- PASS: TestConvertDuration/invalid_duration (0.00s)
    --- PASS: TestConvertDuration/JSON_number (0.00s)
    --- PASS: TestConvertDuration/JSON_bool (0.00s)
=== RUN   TestDefaultHTTPError
=== RUN   TestDefaultHTTPError/0
=== RUN   TestDefaultHTTPError/1
=== RUN   TestDefaultHTTPError/2
=== RUN   TestDefaultHTTPError/3
=== RUN   TestDefaultHTTPError/4
--- PASS: TestDefaultHTTPError (0.00s)
    --- PASS: TestDefaultHTTPError/0 (0.00s)
    --- PASS: TestDefaultHTTPError/1 (0.00s)
    --- PASS: TestDefaultHTTPError/2 (0.00s)
    --- PASS: TestDefaultHTTPError/3 (0.00s)
    --- PASS: TestDefaultHTTPError/4 (0.00s)
=== RUN   TestForwardResponseStream
=== RUN   TestForwardResponseStream/encoding
=== RUN   TestForwardResponseStream/empty
=== RUN   TestForwardResponseStream/error
    handler_test.go:121: checking error encodings
=== RUN   TestForwardResponseStream/stream_error
=== RUN   TestForwardResponseStream/response_body_stream_case
=== RUN   TestForwardResponseStream/response_body_stream_error_case
--- PASS: TestForwardResponseStream (0.00s)
    --- PASS: TestForwardResponseStream/encoding (0.00s)
    --- PASS: TestForwardResponseStream/empty (0.00s)
    --- SKIP: TestForwardResponseStream/error (0.00s)
    --- PASS: TestForwardResponseStream/stream_error (0.00s)
    --- PASS: TestForwardResponseStream/response_body_stream_case (0.00s)
    --- PASS: TestForwardResponseStream/response_body_stream_error_case (0.00s)
=== RUN   TestForwardResponseStreamCustomMarshaler
=== RUN   TestForwardResponseStreamCustomMarshaler/encoding
=== RUN   TestForwardResponseStreamCustomMarshaler/empty
=== RUN   TestForwardResponseStreamCustomMarshaler/error
    handler_test.go:252: checking erorr encodings
=== RUN   TestForwardResponseStreamCustomMarshaler/stream_error
    handler_test.go:252: checking erorr encodings
--- PASS: TestForwardResponseStreamCustomMarshaler (0.00s)
    --- PASS: TestForwardResponseStreamCustomMarshaler/encoding (0.00s)
    --- PASS: TestForwardResponseStreamCustomMarshaler/empty (0.00s)
    --- SKIP: TestForwardResponseStreamCustomMarshaler/error (0.00s)
    --- SKIP: TestForwardResponseStreamCustomMarshaler/stream_error (0.00s)
=== RUN   TestForwardResponseMessage
=== RUN   TestForwardResponseMessage/standard_marshaler
=== RUN   TestForwardResponseMessage/httpbody_marshaler
=== RUN   TestForwardResponseMessage/custom_marshaler
--- PASS: TestForwardResponseMessage (0.00s)
    --- PASS: TestForwardResponseMessage/standard_marshaler (0.00s)
    --- PASS: TestForwardResponseMessage/httpbody_marshaler (0.00s)
    --- PASS: TestForwardResponseMessage/custom_marshaler (0.00s)
=== RUN   TestHTTPBodyContentType
--- PASS: TestHTTPBodyContentType (0.00s)
=== RUN   TestHTTPBodyMarshal
--- PASS: TestHTTPBodyMarshal (0.00s)
=== RUN   TestJSONBuiltinMarshal
--- PASS: TestJSONBuiltinMarshal (0.00s)
=== RUN   TestJSONBuiltinMarshalField
--- PASS: TestJSONBuiltinMarshalField (0.00s)
=== RUN   TestJSONBuiltinMarshalFieldKnownErrors
--- PASS: TestJSONBuiltinMarshalFieldKnownErrors (0.00s)
=== RUN   TestJSONBuiltinsnmarshal
--- PASS: TestJSONBuiltinsnmarshal (0.00s)
=== RUN   TestJSONBuiltinUnmarshalField
--- PASS: TestJSONBuiltinUnmarshalField (0.00s)
=== RUN   TestJSONBuiltinUnmarshalFieldKnownErrors
--- PASS: TestJSONBuiltinUnmarshalFieldKnownErrors (0.00s)
=== RUN   TestJSONBuiltinEncoder
--- PASS: TestJSONBuiltinEncoder (0.00s)
=== RUN   TestJSONBuiltinEncoderFields
--- PASS: TestJSONBuiltinEncoderFields (0.00s)
=== RUN   TestJSONBuiltinDecoder
--- PASS: TestJSONBuiltinDecoder (0.00s)
=== RUN   TestJSONBuiltinDecoderFields
--- PASS: TestJSONBuiltinDecoderFields (0.00s)
=== RUN   TestJSONPbMarshal
=== RUN   TestJSONPbMarshal/0
=== RUN   TestJSONPbMarshal/1
=== RUN   TestJSONPbMarshal/2
=== RUN   TestJSONPbMarshal/3
=== RUN   TestJSONPbMarshal/4
--- PASS: TestJSONPbMarshal (0.00s)
    --- PASS: TestJSONPbMarshal/0 (0.00s)
    --- PASS: TestJSONPbMarshal/1 (0.00s)
    --- PASS: TestJSONPbMarshal/2 (0.00s)
    --- PASS: TestJSONPbMarshal/3 (0.00s)
    --- PASS: TestJSONPbMarshal/4 (0.00s)
=== RUN   TestJSONPbMarshalFields
--- PASS: TestJSONPbMarshalFields (0.00s)
=== RUN   TestJSONPbUnmarshal
--- PASS: TestJSONPbUnmarshal (0.00s)
=== RUN   TestJSONPbUnmarshalFields
--- PASS: TestJSONPbUnmarshalFields (0.00s)
=== RUN   TestJSONPbEncoder
--- PASS: TestJSONPbEncoder (0.00s)
=== RUN   TestJSONPbEncoderFields
--- PASS: TestJSONPbEncoderFields (0.00s)
=== RUN   TestJSONPbDecoder
--- PASS: TestJSONPbDecoder (0.00s)
=== RUN   TestJSONPbDecoderFields
--- PASS: TestJSONPbDecoderFields (0.00s)
=== RUN   TestJSONPbDecoderUnknownField
--- PASS: TestJSONPbDecoderUnknownField (0.00s)
=== RUN   TestJSONPbUnmarshalNullField
--- PASS: TestJSONPbUnmarshalNullField (0.00s)
=== RUN   TestJSONPbMarshalResponseBodies
=== RUN   TestJSONPbMarshalResponseBodies/0
=== RUN   TestJSONPbMarshalResponseBodies/1
=== RUN   TestJSONPbMarshalResponseBodies/2
=== RUN   TestJSONPbMarshalResponseBodies/3
=== RUN   TestJSONPbMarshalResponseBodies/4
=== RUN   TestJSONPbMarshalResponseBodies/5
=== RUN   TestJSONPbMarshalResponseBodies/6
=== RUN   TestJSONPbMarshalResponseBodies/7
=== RUN   TestJSONPbMarshalResponseBodies/8
=== RUN   TestJSONPbMarshalResponseBodies/9
=== RUN   TestJSONPbMarshalResponseBodies/10
=== RUN   TestJSONPbMarshalResponseBodies/11
=== RUN   TestJSONPbMarshalResponseBodies/12
--- PASS: TestJSONPbMarshalResponseBodies (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/0 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/1 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/2 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/3 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/4 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/5 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/6 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/7 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/8 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/9 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/10 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/11 (0.00s)
    --- PASS: TestJSONPbMarshalResponseBodies/12 (0.00s)
=== RUN   TestProtoMarshalUnmarshal
--- PASS: TestProtoMarshalUnmarshal (0.00s)
=== RUN   TestProtoEncoderDecodert
--- PASS: TestProtoEncoderDecodert (0.00s)
=== RUN   TestMarshalerForRequest
--- PASS: TestMarshalerForRequest (0.00s)
=== RUN   TestMuxServeHTTP
=== RUN   TestMuxServeHTTP/0
=== RUN   TestMuxServeHTTP/1
=== RUN   TestMuxServeHTTP/2
=== RUN   TestMuxServeHTTP/3
=== RUN   TestMuxServeHTTP/4
=== RUN   TestMuxServeHTTP/5
=== RUN   TestMuxServeHTTP/6
=== RUN   TestMuxServeHTTP/7
=== RUN   TestMuxServeHTTP/8
=== RUN   TestMuxServeHTTP/9
=== RUN   TestMuxServeHTTP/10
=== RUN   TestMuxServeHTTP/11
=== RUN   TestMuxServeHTTP/12
=== RUN   TestMuxServeHTTP/13
=== RUN   TestMuxServeHTTP/14
=== RUN   TestMuxServeHTTP/15
=== RUN   TestMuxServeHTTP/16
=== RUN   TestMuxServeHTTP/17
=== RUN   TestMuxServeHTTP/18
=== RUN   TestMuxServeHTTP/19
=== RUN   TestMuxServeHTTP/20
--- PASS: TestMuxServeHTTP (0.00s)
    --- PASS: TestMuxServeHTTP/0 (0.00s)
    --- PASS: TestMuxServeHTTP/1 (0.00s)
    --- PASS: TestMuxServeHTTP/2 (0.00s)
    --- PASS: TestMuxServeHTTP/3 (0.00s)
    --- PASS: TestMuxServeHTTP/4 (0.00s)
    --- PASS: TestMuxServeHTTP/5 (0.00s)
    --- PASS: TestMuxServeHTTP/6 (0.00s)
    --- PASS: TestMuxServeHTTP/7 (0.00s)
    --- PASS: TestMuxServeHTTP/8 (0.00s)
    --- PASS: TestMuxServeHTTP/9 (0.00s)
    --- PASS: TestMuxServeHTTP/10 (0.00s)
    --- PASS: TestMuxServeHTTP/11 (0.00s)
    --- PASS: TestMuxServeHTTP/12 (0.00s)
    --- PASS: TestMuxServeHTTP/13 (0.00s)
    --- PASS: TestMuxServeHTTP/14 (0.00s)
    --- PASS: TestMuxServeHTTP/15 (0.00s)
    --- PASS: TestMuxServeHTTP/16 (0.00s)
    --- PASS: TestMuxServeHTTP/17 (0.00s)
    --- PASS: TestMuxServeHTTP/18 (0.00s)
    --- PASS: TestMuxServeHTTP/19 (0.00s)
    --- PASS: TestMuxServeHTTP/20 (0.00s)
=== RUN   TestDefaultHeaderMatcher
=== RUN   TestDefaultHeaderMatcher/permanent_HTTP_header_should_return_prefixed
=== RUN   TestDefaultHeaderMatcher/key_prefixed_with_MetadataHeaderPrefix_should_return_without_the_prefix
=== RUN   TestDefaultHeaderMatcher/non-permanent_HTTP_header_key_without_prefix_should_not_return
--- PASS: TestDefaultHeaderMatcher (0.00s)
    --- PASS: TestDefaultHeaderMatcher/permanent_HTTP_header_should_return_prefixed (0.00s)
    --- PASS: TestDefaultHeaderMatcher/key_prefixed_with_MetadataHeaderPrefix_should_return_without_the_prefix (0.00s)
    --- PASS: TestDefaultHeaderMatcher/non-permanent_HTTP_header_key_without_prefix_should_not_return (0.00s)
=== RUN   TestServeMux_HandlePath
=== RUN   TestServeMux_HandlePath/Test_route_/
=== RUN   TestServeMux_HandlePath/Simple_Endpoint
=== RUN   TestServeMux_HandlePath/Complex_Endpoint
=== RUN   TestServeMux_HandlePath/Wildcard_Endpoint
=== RUN   TestServeMux_HandlePath/Invalid_Endpoint
--- PASS: TestServeMux_HandlePath (0.00s)
    --- PASS: TestServeMux_HandlePath/Test_route_/ (0.00s)
    --- PASS: TestServeMux_HandlePath/Simple_Endpoint (0.00s)
    --- PASS: TestServeMux_HandlePath/Complex_Endpoint (0.00s)
    --- PASS: TestServeMux_HandlePath/Wildcard_Endpoint (0.00s)
    --- PASS: TestServeMux_HandlePath/Invalid_Endpoint (0.00s)
=== RUN   TestPopulateParameters
=== RUN   TestPopulateParameters/0
=== RUN   TestPopulateParameters/1
=== RUN   TestPopulateParameters/2
=== RUN   TestPopulateParameters/3
=== RUN   TestPopulateParameters/4
=== RUN   TestPopulateParameters/5
=== RUN   TestPopulateParameters/6
=== RUN   TestPopulateParameters/7
=== RUN   TestPopulateParameters/8
=== RUN   TestPopulateParameters/9
=== RUN   TestPopulateParameters/10
=== RUN   TestPopulateParameters/11
=== RUN   TestPopulateParameters/12
--- PASS: TestPopulateParameters (0.01s)
    --- PASS: TestPopulateParameters/0 (0.00s)
    --- PASS: TestPopulateParameters/1 (0.00s)
    --- PASS: TestPopulateParameters/2 (0.00s)
    --- PASS: TestPopulateParameters/3 (0.00s)
    --- PASS: TestPopulateParameters/4 (0.00s)
    --- PASS: TestPopulateParameters/5 (0.00s)
    --- PASS: TestPopulateParameters/6 (0.00s)
    --- PASS: TestPopulateParameters/7 (0.00s)
    --- PASS: TestPopulateParameters/8 (0.00s)
    --- PASS: TestPopulateParameters/9 (0.00s)
    --- PASS: TestPopulateParameters/10 (0.00s)
    --- PASS: TestPopulateParameters/11 (0.00s)
    --- PASS: TestPopulateParameters/12 (0.00s)
=== RUN   TestPopulateParametersWithFilters
--- PASS: TestPopulateParametersWithFilters (0.00s)
=== RUN   TestPopulateQueryParametersWithInvalidNestedParameters
--- PASS: TestPopulateQueryParametersWithInvalidNestedParameters (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/runtime       (cached)
?       github.com/grpc-ecosystem/grpc-gateway/v2/runtime/internal/examplepb    [no test files]
=== RUN   TestMaxCommonPrefix
--- PASS: TestMaxCommonPrefix (0.00s)
=== RUN   TestAdd
--- PASS: TestAdd (0.00s)
PASS
ok      github.com/grpc-ecosystem/grpc-gateway/v2/utilities     (cached)
[root@406a14c090d5 /]#

Reply:
"master" is a moving target. It keeps changing and at some point of time the script may not work with "master". Hence it is better to set the default to the exact version which has been tested.